### PR TITLE
multiple fixes

### PR DESCRIPTION
**Pull request by jogibear9988**

### CODE REVIEW

**Comment by mar10** [comment]
What do you think about about having an option `serializeClipboardData` instead, that defaults to true.
`false` prevents serialization altogether.
A callback function may be passed, that receives a WunderbaumNode (orl array thereof) and returns a string.
That would allow to serialize a custom set of attributes...

**Comment by jogibear9988** [comment]
it doesn't matter for me how it's done, i only need to be able to disable it

**Comment by jogibear9988** [comment]
this pull also solves #39 & #41

**Comment by mar10** [comment]
Thanks for contributing 👍 

I would merge the serializeClipboardData feature immediately, but need some details and spec about svg support.
(Let's keep PRs separate per topic next time please ;-)

**Comment by jogibear9988** [comment]
If you use img instead of css background, the drag over the img node drags the image instead of the treenode

**Comment by jogibear9988** [comment]
I thought maybe include some images as default, so users don‘t need bootstrap or another icon lib to have icons from the start

**Comment by jogibear9988** [comment]
I directly link to svg files from imagemap

**Comment by jogibear9988** [comment]
I worked on more, also on some typeinfos, I'll look to split up this pull req. tomorrow

File: src/types.ts [hunk 1]
```diff
@@ -769,12 +891,21 @@ export type DndOptionsType = {
...
    */
-  dragExpand: null;
+  dragExpand?: null | ((e: WbNodeEventType & { event: DragEvent }) => void);
   /**
    * Callback(targetNode, data)
    * @default null
    */
-  dragDrop: null;
+  drop?:
+    | null
```
Comment (mar10) [inline src/types.ts:900]: small typo here (leading pipe)?

**Comment by mar10** [comment]
Great job with the typings, thank you 👍
Yes , it would be great to split serialize, icon, and typings

**Comment by jogibear9988** [comment]
> Great job with the typings, thank you 👍 Yes , it would be great to split serialize, icon, and typings

I'm little bit busy atm. hopefully I can split it up in the next days

File: src/types.ts (hunk 1 above)
Comment (jogibear9988) [reply src/types.ts:900]: The leading pip is inserted by prettier

**Comment by jogibear9988** [comment]
i'll remove this pull req when i extracted all commits

**Pull request closed without merging**